### PR TITLE
Rename parser wrapper

### LINKED ISSUE #514: Rename `@parity/zombienet-dsl-parser-wrapper` to `@zombienet/dsl-parser-wrapper`?

### PR DESCRIPTION
**Pull request by pepoviola**
close #514 

Rename wrapper and update version.

### CODE REVIEW

File: crates/parser-wrapper/Cargo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [package]
-name = "zombienet-dsl-parser-wrapper"
+name = "dsl-parser-wrapper"
 version = "0.1.6"
```
Comment (wirednkod) [inline crates/parser-wrapper/Cargo.toml:3]: Lets bump the version as well to 0.1.7:
```suggestion
version = "0.1.7"
```

File: javascript/packages/cli/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@parity/zombienet",
-  "version": "1.3.12",
+  "version": "1.3.13",
```
Comment (wirednkod) [inline javascript/packages/cli/package.json:3]: Lets NOT bump this version just yet

File: javascript/packages/cli/package.json [hunk 3]
```diff
@@ -50,7 +50,7 @@
     "url": "https://github.com/paritytech/zombienet/issues"
   },
   "dependencies": {
-    "@parity/zombienet-dsl-parser-wrapper": "^0.1.6",
+    "@zombienet/zombienet-dsl-parser-wrapper": "^0.1.6",
```
Comment (wirednkod) [inline javascript/packages/cli/package.json:53]: I think this should be:
```suggestion
    "@zombienet/dsl-parser-wrapper": "^0.1.6",
```

**Review by wirednkod — APPROVED** [review]

File: javascript/packages/cli/package.json (hunk 3 above)
Comment (pepoviola) [reply javascript/packages/cli/package.json]: Yes, typo already fixed.

File: javascript/packages/cli/package.json (hunk 2 above)
Comment (pepoviola) [reply javascript/packages/cli/package.json:3]: Oks, you want to wait to your pr?

File: crates/parser-wrapper/Cargo.toml (hunk 1 above)
Comment (pepoviola) [reply crates/parser-wrapper/Cargo.toml:3]: Is not really needed, since is already pushed to `npm` as `0.1.6` (firs version of this module).

**Pull request merged**